config/examples/Creality/Ender-3 Pro/BigTreeTech SKR Mini E3 2.0/Configuration_adv.h

```diff
@@ -1045,7 +1045,9 @@
   #define RESTORE_LEVELING_AFTER_G35    // Enable to restore leveling setup after operation
   //#define REPORT_TRAMMING_MM          // Report Z deviation (mm) for each point relative to the first
 
-  //#define ASSISTED_TRAMMING_WIZARD    // Add a Tramming Wizard to the LCD menu
+  #if EITHER(HAS_MARLINUI_MENU, HAS_TFT_LVGL_UI)
+    //#define ASSISTED_TRAMMING_WIZARD    // Add a Tramming Wizard to the LCD menu
+  #endif
```

Comment:
To repeat what I said in https://github.com/MarlinFirmware/Configurations/pull/872, this change should not go here. Please add a sanity check upstream in Marlin to check for this invalid config instead of adding it to this single config.